src/build/loaders/workbook-json-loader.js

```diff
@@ -0,0 +1,56 @@
+const path = require("path");
+
+/**
+ *
+ * @param {string} content
+ * @returns {string}
+ */
+function insertToc(content) {
+  let toc = '<h2 class="toc">&nbsp;</h2><ul class="toc">';
+  const regexp = /<h2 id="(?<id>[^"]*)"[^>]*>(?<title>.+?)<\/h2>/g;
+  const matches = content.matchAll(regexp);
+  for (const match of matches) {
+    const { id, title } = match.groups || {};
+    toc += `<li><a href="#${id}">${title}</a></li>`;
+  }
+  toc += '</ul><div class="pb"></div>';
+
+  return content.replaceAll("<!-- toc -->", toc);
+}
+
+/**
+ * @param {string} labsJsonFileContent
```

Comment:
```suggestion
 * @param {string} workbookJsonFileContent
```

Pour être cohérent avec le nom du fichier, à renommer partout.